### ISSUE TITLE
Cannot parse selector error with dynamic state selector

### ISSUE THREAD

**Issue by pvdlg** [styled-components/stylelint-processor-styled-components#256]
When using a JS variable for a `hover`, `active`, `focus` selector the parser fails and throw the error `Cannot parse selector`.

### System:
 - OS: macOS 10.14.3
 - CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
 - Memory: 237.40 MB / 16.00 GB
 - Shell: 3.2.57 - /bin/bash
### Binaries:
 - Node: 11.4.0 - /usr/local/bin/node
 - Yarn: 1.12.3 - /usr/local/bin/yarn
 - npm: 6.7.0 - /usr/local/bin/npm
### npmPackages:
 - styled-components: ^4.1.3 => 4.1.3
 - stylelint: ^9.10.1 => 9.10.1
 - stylelint-config-recommended: ^2.1.0 => 2.1.0
 - stylelint-config-styled-components: ^0.1.1 => 0.1.1
 - stylelint-processor-styled-components: ^1.5.2 => 1.5.2

### .stylelintrc:
```json
{
    "processors": [
      "stylelint-processor-styled-components"
    ],
    "extends": [
      "stylelint-config-recommended",
      "stylelint-config-styled-components"
    ]
  }
```

## Reproduction

With the following code
```js
import {css} from 'styled-components';

const stateStyle = state => css`
  :${state} div,
    color: ${props => getStateColor(state)};
  }
`;
```
stylelint throws `Cannot parse selector`.

## Expected Behavior

The code should be parsed properly.

**Comment by chinesedfan** [comment]
The processor can not always parse interpolations with right things. But you can use [interpolation-tagging](https://www.styled-components.com/docs/tooling#interpolation-tagging) to help it. If you have ideas to make it more intelligent, feel free to send a PR or share your solution by an new issue.

```
import {css} from 'styled-components';

// ps. I fixed your sample `div, ` to `div {`
const stateStyle = state => css`
  :${state /* sc-custom 'hover' */} div {
    color: ${props => getStateColor(state)};
  }
`;
```
